### PR TITLE
Get the 2018 train rolling 🚂

### PR DESCRIPTION
**Pull request by josh-**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,33 +7,14 @@
...
-              <span>Sydney</span>
-              <span class="light">19 &mdash; 20 August</span>
-            </h2>
-            <p>
-              <a class="uh_btn" href="{{ site.github.url }}/sydney/">Register Now</a>
-            </p>
-          </div>
-        </div>
+          <h2>We'll be back</h2>
+          <h2><span class="light">UNIHACK is coming back bigger & better in 2018</span></h2>
```
Comment (terencehuynh) [inline index.html]: `&amp;` instead of `&`

File: sydney/index.html [hunk 2]
```diff
@@ -5,109 +5,24 @@
...
-  </div>
-</div>
-
-{% include event/sponsor_partner.html %}
-
-{% include event/faq.html %}
-
-{% include event/about.html %}
-
-{% include event/society.html society="suits" %}
```
Comment (terencehuynh) [inline sydney/index.html]: Let's leave this bit in. (the society bit)

File: melbourne/index.html [hunk 3]
```diff
@@ -5,118 +5,24 @@
...
-  </div>
-</div>
-
-{% include event/sponsor_partner.html %}
-
-{% include event/faq.html %}
-
-{% include event/about.html %}
-
-{% include event/society.html society="wired" css="no-border" %}
```
Comment (terencehuynh) [inline melbourne/index.html]: Let's leave this bit in. (the society bit)

**Comment by josh-** [comment]
Thanks @terencehuynh, should be good to go 🚀

**Pull request merged**